### DISCUSSION TITLE
Maybe release binary?

### DISCUSSION THREAD

**Discussion by Shuzhengz** [JakeRoggenbuck/auto-clock-speed#482] [Q&A]
maybe we should release a compiled binary (and possibly publish to a package manager) because the build time is suer long right now and might not be good for people using old systems.

**Comment by JakeRoggenbuck** [comment]
We have a PKGBUILD in main and I have an aur account ready to publish. I think the idea was to not publish until alpha release. I think we are really close to alpha version, so we can start publishing there soon. I think adding a compiled binary to the release each time would be pretty cool.

**Comment by Shuzhengz** [comment]
oh cool
